docs/integrations/sources/cockroachdb.md

```diff
@@ -0,0 +1,92 @@
+# CockroachDb
+
+## Overview
+
+The CockroachDB source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+### Resulting schema
+
+The CockroachDb source does not alter the schema present in your database. Depending on the destination connected to this source, however, the schema may be altered. See the destination's documentation for more details.
+
+### Data type mapping
+
+CockroachDb data types are mapped to the following data types when synchronizing data:
+
+| CockroachDb Type | Resulting Type | Notes |
+| :--- | :--- | :--- |
+| `bigint` | integer |  |
+| `bit` | boolean |  |
+| `boolean` | boolean |  |
+| `character` | string |  |
+| `character varying` | string |  |
+| `date` | string |  |
+| `double precision` | string |  |
+| `enum` | number |  |
+| `inet` | string |  |
+| `int` | integer |  |
+| `json` | string |  |
+| `jsonb` | string |  |
+| `numeric` | number |  |
+| `smallint` | integer |  |
+| `text` | string |  |
+| `time with timezone` | string | may be written as a native date type depending on the destination |
+| `time without timezone` | string | may be written as a native date type depending on the destination |
+| `timestamp with timezone` | string | may be written as a native date type depending on the destination |
+| `timestamp without timezone` | string | may be written as a native date type depending on the destination |
+| `uuid` | string |  |
+
+**Note:** arrays for all the above types as well as custom types are supported, although they may be de-nested depending on the destination. Byte arrays are currently unsupported.
```

Comment:
what happens to byte arrays? does the connector just break?

Reply:
My bad, I copy-pasted the template from the postgres source and probably missed to clean this Note. Byte arrays also works, added additional test for it to comprehensive tests. Thanks